starfinder/db/models.py

```diff
@@ -179,26 +179,43 @@ class Equipment(db_engine.Model, ModelBase, HasId):
 #                                      nullable=False)
 
 
-# class RangedWeapon(db_engine.Model, ModelBase, HasId):
-#     attributes = db_engine.Column(db_engine.JSON("ranged_weapons.attributes"),
-#                                      nullable=False)
+class RangedWeapon(db_engine.Model, ModelBase, HasId):
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    category = db_engine.Column(db_engine.Integer(), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=False)
+    price = db_engine.Column(db_engine.Integer(), nullable=False)
+    damage = db_engine.Column(db_engine.String(64), nullable=False)
+    range_atk = db_engine.Column(db_engine.String(64), nullable=False)
+    critical = db_engine.Column(db_engine.String(64), nullable=False)
+    capacity = db_engine.Column(db_engine.String(64), nullable=False)
+    usage = db_engine.Column(db_engine.Integer(), nullable=False)
+    bulk = db_engine.Column(db_engine.String(64), nullable=False)
+    special = db_engine.Column(db_engine.String(64), nullable=False)
+    description = db_engine.Column(db_engine.String(64), nullable=False)
+
 
 
 # ------------
 # Class Info Classes 
 # ------------
 
 class Class(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
```

Comment:
good call! I forgot about this, but anywhere that we require _incrementing_ `id`s, we need to change the value here to `Integer`. Thanks for catching that!